sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<!-- Copyright 2018 Tecnativa - Carlos Dauden
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<record id="view_sales_order_line_input_tree" model="ir.ui.view">
+    <field name="name">sale.order.line.input.tree.</field>
+    <field name="model">sale.order.line</field>
+    <field name="priority">20</field>
+    <field name="arch" type="xml">
+        <tree string="Sales Orders Lines"
+              decoration-info="invoice_status=='to invoice'" decoration-warning="state=='draft'" decoration-muted="state=='cancel'" decoration-success="state=='done'"
+              editable="top" create="true">
+            <field name="order_id" string="Order"
+                   required="0" placeholder="New"
+                   domain="[('state', 'not in', ('done', 'cancel'))]"
+                   options='{"no_open": True}'/>
+            <button name="action_sale_order_form" type="object" icon="fa-external-link" class="oe_stat_button"/>
```

Comment:
Should this go in popup?

Reply:
The grace of the button is to open it inline